source/memory.tex

```diff
@@ -5401,8 +5402,9 @@
 even though they use the same static allocator type.
 
 \pnum
-All specializations of class template \tcode{pmr::polymorphic_allocator}
-meet the allocator completeness requirements\iref{allocator.requirements.completeness}.
+Every specialization of class template \tcode{pmr::polymorphic_allocator}
```

Comment:
I'm a little uncomfortable with saying "every specialization ... if ..." since it's not every specialization.

How about:

```suggestion
A specialization of class template \tcode{pmr::polymorphic_allocator}
```